setup.py

```diff
@@ -171,6 +230,63 @@ def get_extensions():
             )
         )
 
+    # ------------------- Torchvision extra extensions ------------------------
+    vision_include = os.environ.get('TORCHVISION_INCLUDE', None)
+    vision_library = os.environ.get('TORCHVISION_LIBRARY', None)
+    vision_include = (vision_include.split(os.pathsep)
+                      if vision_include is not None else [])
+    vision_library = (vision_library.split(os.pathsep)
+                      if vision_library is not None else [])
+    include_dirs += vision_include
+    library_dirs = vision_library
+
+    # Image reading extension
+    image_macros = []
+    image_include = [extensions_dir]
+    image_library = []
+    image_link_flags = []
+
+    # Locating libPNG
+    libpng = distutils.spawn.find_executable('libpng-config')
+    png_found = libpng is not None
+    image_macros += [('PNG_FOUND', str(int(png_found)))]
+    print('PNG found: {0}'.format(png_found))
+    if png_found:
+        png_version = subprocess.run([libpng, '--version'],
+                                     stdout=subprocess.PIPE)
+        png_version = png_version.stdout.strip().decode('utf-8')
+        print('libpng version: {0}'.format(png_version))
+        png_version = parse_version(png_version)
+        if png_version >= parse_version("1.6.0"):
+            print('Building torchvision with PNG image support')
+            png_lib = subprocess.run([libpng, '--libdir'],
```

Comment:
Ok, I saw this. Unluckily, on Windows, libpng-config doesn't exist. But there is a CMake configuration. Reading that file, it seems that you'll only need to link against `libpng.lib` and set the corresponding include directory.